sopel/tools/calculation.py

```diff
@@ -42,7 +42,11 @@ def __init__(
         self.binary_ops = bin_ops or {}
         self.unary_ops = unary_ops or {}
 
-    def __call__(self, expression_str: str, timeout: float = 5.0):
+    def __call__(
+        self,
+        expression_str: str,
+        timeout: float = 5.0,
+    ) -> int | float:
```

Comment:
IIRC, `int | float` is redundant because `float` can annotate both per [PEP 484](https://peps.python.org/pep-0484/#the-numeric-tower)

I'm okay with explicitly annotating it as a way to document the specific type that will be present but would personally prefer an alias for the union in that case, as a place to leave a comment about why we don't do it the "usual" way.

Reply:
Good call.